docs/faq.rst

```diff
@@ -1,60 +1,743 @@
 FAQ
-===
+=====
 
-This is the FAQ page. Feel free to contribute any FAQs that you think are missing. 
+This is the FAQ page. Feel free to contribute any FAQs that you think are missing.
```

Comment:
@SteveGilvarry I'm not sure if we want this to text to show as a TOC element. It shows on the right. The TOC is shown on the left sidebar and comprises only of headings and subheadings (if you expand the headings). I'm following that convention, so it leaves the TOC list easy to navigate